lib/fetchers/cdl_fetcher.py

```diff
@@ -1,187 +1,34 @@
-import json
-from dplaingestion.fetchers.fetcher import *
+from dplaingestion.fetchers.absolute_url_fetcher import *
 
-class CDLFetcher(Fetcher):
-    def __init__(self, profile, uri_base, config_file):
-        super(CDLFetcher, self).__init__(profile, uri_base, config_file)
-        self.total_records = None
-        self.endpoint_url_params = profile.get("endpoint_url_params")
-
-    def extract_content(self, content, url):
-        error = None
-        try:
-            parsed_content = json.loads(content)
-        except Exception, e:
-            error = "Error parsing content from URL %s: %s" % (url, e)
-            return error, content
 
-        if not self.total_records:
-            total_records_prop = "total_rows"
-            self.total_records = getprop(parsed_content, total_records_prop)
+class CDLFetcher(AbsoluteURLFetcher):
 
-        if parsed_content is None:
-            error = "Error, there is no content from " \
-                    "URL %s" % url
-        return error, parsed_content
+    def __init__(self, profile, uri_base, config_file):
+        super(CDLFetcher, self).__init__(profile, uri_base, config_file)
+        token = self.config.get("APITokens", "CDL")
+        authorization = \
+            self.http_headers["X-Authentication-Token"].format(token)
```

Comment:
I unfortunately was biting off what was there for NYPL. I agree it didn't make much sense, but I think it has something to do with the way http_headers is set by whatever stuff Ingestion1 does to automatically set these properties and that having been written before the idea of auth tokens being passed in from akara.ini started happening. 

You can see what I mean here: https://github.com/dpla/ingestion/blob/develop/lib/fetchers/nypl_fetcher.py#L7-L8